ci/pipeline.yml

```diff
@@ -95,6 +95,20 @@ jobs:
       AWS_DEFAULT_REGION: {{aws-region}}
       GATEWAY_HOST: {{gateway-host}}
 
+- name: prometheus-down-check
+  serial_groups: [production]
+  plan:
+  - aggregate:
+    - get: prometheus-check-timer
+      trigger: true
+    - get: prometheus-config
+      resource: prometheus-config
+  - task: prometheus-down
+    file: prometheus-config/ci/prometheus-down.yml
+    params:
+      AWS_DEFAULT_REGION: {{aws-region}}
```

Comment:
Looks like these don't get used.

Reply:
Yeah, I was wondering about that.  I mainly just cut/pasted from the others.  I didn't know if those were needed for some other reason.